ls1intum/Artemis
# `Development`: Add database tests run configurations and docs

### Checklist
#### General
- [x] This is a small issue that I tested locally and was confirmed by another developer **locally and on docs page**.

### Motivation and Context
Despite database server tests for MySQL and Postgres being important part of our CI, Artemis documentation did not included any information on how to start them. IntelliJ run configurations for those tests was also not included  in the repository. This confused some students.

### Description
This PR adds: 
- a documentation on how to start database server tests for MySQL and Postgres
   - docs can be found under _Contributor Guide/Setup Guide/Running database tests locally_ 
- IntelliJ run configurations for starting MySQL and Postgres tests

### Steps for Testing
- read the docs and verify if they are clear (you can download docs as HTML from the artefact produced by the latest _Documentation/docs_ GitHub Action run)
- checkout the branch and run database tests from IntelliJ

### Review Progress
#### Code Review
